app/models/github_fetcher/null_issue_response.rb

```diff
@@ -0,0 +1,15 @@
+module GithubFetcher
+  class NullIssueResponse
```

Comment:
OK nvm I understand now, I didn't realize "Issue" was for a single Issue, not an array of them.

Maybe we need a class like: 

```ruby
module GithubFetcher
   class Issues
     include Enumerable
   end
end
```

See [here](https://stackoverflow.com/questions/7220634/how-do-i-use-the-enumerable-mixin-in-my-class).